docs/user/azure/install_upi.md

```diff
@@ -288,11 +289,17 @@ Create the deployment using the `az` client:
 
 ```sh
 export VHD_BLOB_URL=`az storage blob url --account-name ${CLUSTER_NAME}sa --account-key $ACCOUNT_KEY -c vhd -n "rhcos.vhd" -o tsv`
+export RELEASE_VERSION=`openshift-install coreos print-stream-json | jq -r '.architectures.$OCP_ARCH."rhel-coreos-extensions"."azure-disk".release'`
+export STORAGE_ACCOUNT_ID=`az storage account show -g ${RESOURCE_GROUP} --name ${CLUSTER_NAME}sa --query id -o tsv`
```

Comment:
| jq -r ".architectures.$OCP_ARCH.\"rhel-coreos-extensions\".\"azure-disk\".release"

Reply:
Tks, I need to update this. This is how I did it in CI https://github.com/openshift/release/pull/34678/files#diff-5483a633ffe74be033f28def4e9b0c9adbfb0ec96b53e43ef314e22672b64ae9R162

Reply:
Fixed